### DISCUSSION TITLE
Installation process on OSX

### DISCUSSION THREAD

**Discussion by maxgawason** [opcm/pcm#352] [Q&A]
I have been trying to follow the install instructions for Mac but I have gotten stuck trying to run pcm. Since the automatic install was initially failing, I attempted the manual install. Since /usr is now a read-only file on Mac, I added libPcmMsr.dylib, MSRKernel.h and MSRAccessorPublic directly to my path. However, when I attempt to run ./pcm.x it continues to look in /usr/lib for libPcmMsr.dylib. 
```
$ ../pcm.x utility
dyld: Library not loaded: /usr/lib/libPcmMsr.dylib
  Referenced from: /Users/maxgawason/cs598/graphOrder/cacheStuff/pcm/MacMSRDriver/../pcm.x
  Reason: image not found
Abort trap: 6
```
Is OSX no longer supported now that /usr is not writable? How can I fix this?

I am on macOS Big Sur 11.6

**Comment by rdementi** [comment]
We don't have a system to test right now. One can check if  /usr/lib is hard coded in PCM and change these paths.
